Sources/WhisperKit/Core/WhisperKit.swift

```diff
@@ -6,7 +6,7 @@ import AVFoundation
 import CoreML
 import Foundation
 import Hub
-import TensorUtils
+//import TensorUtils
```

Comment:
Unrelated to the Tokenizers/Hub changes, we may need to expose this as part of a new smallish target.